### PR TITLE
fix(cloud-service-card): fix cloud-service-card text overflow issue

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [x] Not that difficult
- [ ] Approved feature branch merge to master

### Description
[Quest-859](https://pyengine.atlassian.net/browse/QUEST-859)
Apply multiple ellipsis style to `cloud-service-list-card`.

Before
image

After
image

### CODE REVIEW

**Review by skdud4659 — APPROVED** [review]
LGTM~

File: apps/web/src/services/asset-inventory/cloud-service/modules/cloud-service-list/CloudServiceListCard.vue [hunk 1]
```diff
@@ -190,15 +190,20 @@ export default {
...
+                    max-width: 90%;
                     font-weight: 400;
                     line-height: 1.25;
-                    display: inline-block;
+                    display: -webkit-box;
+                    -webkit-line-clamp: 2;
+                    -webkit-box-orient: vertical;
+                    overflow: hidden;
+                    text-overflow: ellipsis;
+                    white-space: normal;
```
Comment (skdud4659) [inline apps/web/src/services/asset-inventory/cloud-service/modules/cloud-service-list/CloudServiceListCard.vue:201]: nitpick:You can use Tailwind's truncate!

**Review by sulmoJ — APPROVED** [review]
LGTM!!

File: apps/web/src/services/asset-inventory/cloud-service/modules/cloud-service-list/CloudServiceListCard.vue (hunk 1 above)
Comment (piggggggggy) [reply apps/web/src/services/asset-inventory/cloud-service/modules/cloud-service-list/CloudServiceListCard.vue:201]:
```
                    @apply text-sm text-gray-600 truncate;
                    max-width: 90%;
                    font-weight: 400;
                    line-height: 1.25;
                    display: -webkit-box;
                    -webkit-line-clamp: 2;
                    -webkit-box-orient: vertical;
                    //overflow: hidden;
                    //text-overflow: ellipsis;
                    white-space: normal;
                    height: calc(100% + 0.5rem);
```

This works! Thanks for tip

**Pull request merged**